solutions/cses-1653.mdx

```diff
@@ -0,0 +1,141 @@
+---
+id: cses-1653
+source: CSES
+title: Elevator Rides
+author: Sofia Yang
+---
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N\cdot 2^N)$
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class elevators {
+	public static int people, maxWeight;
+	public static int[] weight;
+	public static state[] dp;
```

Comment:
Is there a need for these to be instance variables? If not, it's probably more efficient to just declare them in the main method